_includes/footer/desktop.html

```diff
@@ -0,0 +1,56 @@
+<footer class="footer-desktop" role="contentinfo">
```

Comment:
Whats the reason to to have two different footer versions for mobile and desktop? Shouldn't responsive design work so that you only have one version of the HTML?